src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
 }
+
+$time: 0.6s;
```

Comment:
Better be more specific with names, if it is not too long. Here "transition-time" would be more representative of what this variable exists for